### ISSUE TITLE
Background bug

### ISSUE THREAD

**Issue by terrarier2111** [iceiix/stevenarella#583]
When first joining and then leaving a server, the background of the serverlist becomes the server world (including players etc.)

**Comment by iceiix** [comment]
I suspect this is a _feature_ - but I'm not 100% sure, it predates my involvement with this project. When launched, the background will be a locally generated default world, but when leaving a server it will be the last recently used server. Maybe could revert it to the default world...

**Comment by terrarier2111** [comment]
Yeah i think that would be a good idea or just we may as well remove this feature once and for all and replace it with video playback where we can insert a video of a similar minecraft world which is played in a loop, because of this feature, it takes the game 10 more seconds to start. (without this feature the game starts in under 1 second)

**Comment by iceiix** [comment]
Agreed, if this feature causes startup performance issues, a good argument in favor of removing it. Had no idea it caused that much slowdown (startup takes <1s on my system) but if it does, I wouldn't miss that feature..
